requirements/devenv.in

```diff
@@ -13,12 +13,14 @@
 #        because it is NOT mounted in devel container. Therefore the question is
 #        how to sync??
 
-# formatter
-black
-isort
 # dependency manager
 pip-tools
+
 # version manager
 bump2version
+
 # renaming
 rope
+
+# git hooks. pre-commit fails to install directly!
```

Comment:
this is a bit of a strange comment. not sure I understand what that really means?

Reply:
True, I do not fully understand it myself right now. I will rewrite it.
The thing is as follows: when you try to ``pip install pre-commit``in a fresh venv it warns about incompatible constraints. My guess is that the problem is in the pre-commit's requirements setup. In order to solve the dependencies correctly we use here ``pip-compile``